support-frontend/assets/components/consentBanner/consentBanner.jsx

```diff
@@ -0,0 +1,84 @@
+// @flow
+import React from 'react';
+import type { Dispatch } from 'redux';
+import { connect } from 'react-redux';
+import {
+  OptedIn, Unset,
+} from '../../helpers/tracking/thirdPartyTrackingConsent';
+import {
+  type Action,
+  setTrackingConsent,
+} from '../../helpers/page/commonActions';
+import type { ThirdPartyTrackingConsent } from '../../helpers/tracking/thirdPartyTrackingConsent';
+import ConsentBannerButtonTick from '../svgs/consentBannerButtonTick';
+import ConsentBannerRoundel from '../svgs/consentBannerRoundel';
```

Comment:
FYI! I cheekily made it so you can import actual svgs into webpack (and you get svgo compression for free)

https://github.com/guardian/support-frontend/pull/1628/files#diff-c6c46b0dc809478e653dddee4e6b19dbR8